gists/snippets/react.md

```diff
@@ -117,6 +117,52 @@ Copy the following into your `javascript.json` snippet file:
 			")(${1:MyFormComponent})"
 		],
 		"description": "LP-Form enhanced Redux-Form form component"
+	},
```

Comment:
New Formik snippet. I left the Redux Form snippet in place for existing projects that have to maintain Redux Form forms.